tests/test_camera.py

```diff
@@ -0,0 +1,137 @@
+import unittest
```

Comment:
Should import in this order (with blank lines after each group and alphabetical ordering within groups): 
```
1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.
```
 https://www.python.org/dev/peps/pep-0008/#imports